content/blog/early-adopters-call-sel-botan.md

```diff
@@ -0,0 +1,44 @@
++++
+title = "Call for early adopters for Sel and Botan"
+description = ""
+date = 2024-04-05T16:00:00
+updated = 2024-04-03T16:00:00
+draft = false
+template = "blog/page.html"
+
+[extra]
+lead = "We are calling for early adopters of the sel and botan-low libraries to improve their usability and reliability"
++++
+
+The Haskell Cryptography Group is calling for early adopters of [Sel][sel] and [botan-low], which respectively provide
+reliable and well-documented bindings to [libsodium][libsodium] and [botan].
+
+We want to gather feedback from industrial and hobbyist users on the documentation and ergonomics of these libraries,
+as this will help us serve users better in the future.
+
+In particular, we want to know if the example usages provided in the documentation help, and how they could be improved.
+
+You can use these libraries today by adding them to your cabal file:
+
+```haskell
+botan-low ^>=0.0.1,
+```
+
+and
+
+```haskell
+sel ^>=0.0.1,
+```
+
+For security reasons, the bindings do not bundle the underlying C and C++ libraries, you have to install them yourself.
```

Comment:
I've observed that sometimes people need help finding the installation instructions so I would add something along the lines of 'Please see your specific library's installation instructions for more details.' and link to build instructions directly as well eg https://botan.randombit.net/handbook/building.html and https://libsodium.gitbook.io/doc/installation

Reply:
hmm yeah especially for botan3.